src/usePost.js

```diff
@@ -0,0 +1,41 @@
+import { useState, useEffect } from 'react';
+import axios from 'axios';
+
+const baseURL = 'http://localhost:3000/';
+
+export const usePost = (url, title, body) => {
+    const [success, setSuccess] = useState(false);
+    const [loading, setLoading] = useState(false);
+    const [error, setError] = useState('');
+   
+    const postData = async () => {
+        setError(undefined);
+        setLoading(true);
+
+        try {
+            const dbData = await axios.get(`${baseURL}${url}`);
+            const id = dbData.data.length + 1;
```

Comment:
No need to include `id`. json-server will auto increment the primary key which is `id`